projects/dotcms-field-elements/src/components/dot-form/dot-form.tsx

```diff
@@ -187,4 +200,27 @@ export class DotFormComponent {
             {}
         );
     }
+
+    private uploadFile(event: CustomEvent): void {
+        const uploadService = new DotUploadService();
+        const { name, value } = event.detail;
+        const binary: DotBinaryFile = (event.target as unknown) as DotBinaryFile;
+        if (value) {
```

Comment:
Why this flag here? does this mean that this method could be called with no value in the field?

Reply:
right, if the file is removed or the form gets reset.